src/lib/utils/getPropertyByPath.ts

```diff
@@ -1,5 +1,8 @@
-function getPropertyByPath(object: object, path: string): any {
-  const parts = path.split('.');
+function getPropertyByPath(
+  object: object | undefined,
+  path: string | string[]
+): any {
+  const parts = Array.isArray(path) ? path : path.split('.');
```

Comment:
Do we want to start warning already for string paths?

Reply:
adding a subsequent PR after we've updated the docs